src/validators/RequestSchema.js

```diff
@@ -251,6 +251,7 @@ const UpdateSettingsRequestSchema = Joi.object().keys({
   favicon: Joi.string(),
   "facebook-pixel": Joi.string().allow(""),
   google_analytics: Joi.string().allow(""),
```

Comment:
Style: could we rename `google_analytics` to `google_analytics_ua` for clarity?

Reply:
I'm reluctant to do this because the `google_analytics` param already exists on repos currently - we'd need to run a migration on all existing sites to change `google_analytics` to `google_analytics_ua` in their `config.yml`